segmentation/utils/data_utils_test.py

```diff
@@ -27,27 +28,30 @@ def _create_img_dir(temp_dir, fovs, imgs, img_sub_folder="TIFs"):
 
 
 def test_validate_paths():
-    valid_path = '.'
 
-    valid_parts = [p for p in pathlib.Path(valid_path).resolve().parts]
-    valid_parts[1] = 'not_a_real_directory'
+    valid_path = 'data/example_dataset/label_data'
+
+    valid_parts = [p for p in pathlib.Path(valid_path).parts]
+    valid_parts[0] = 'not_a_real_directory'
 
     starts_out_of_scope = os.path.join(*valid_parts)
 
-    entirely_out_of_scope = '/not_a_real_directory/somewhere_else_not_here'
+    valid_parts[0] = 'data'
+    valid_parts[1] = 'not_a_real_subdirectory'
+    wrong_subdir = os.path.join(*valid_parts)
 
-    nonexistant = './not_a_real_directory'
+    wrong_file = os.path.join(valid_path, 'not_a_real_file.tiff')
 
     data_utils.validate_paths(valid_path)
 
-    with pytest.raises(ValueError, match=r".*starts*"):
+    with pytest.raises(ValueError, match=r".*prefixed*"):
```

Comment:
In line with my comment about the functionality itself, we should check here that the correct element of the path was identified as the typo. So having an example where the last sub_dir is the erroneous one, a sub_dir in the middle, etc.